### PR TITLE
update the sql query to improve the page loading performance

### PR DESCRIPTION
**Pull request by jerrybai2009**

### CODE REVIEW

File: web/app/dao/DatasetsDAO.java [hunk 1]
```diff
@@ -55,11 +55,11 @@
 			"GROUP_CONCAT(IFNULL(u.display_name, '*') ORDER BY o.sort_id ASC SEPARATOR ',') as owner_name, " +
 			"FROM_UNIXTIME(source_created_time) as created, d.source_modified_time, " +
 			"FROM_UNIXTIME(source_modified_time) as modified " +
-			"FROM dict_dataset d " +
+			"FROM ( SELECT * FROM dict_dataset ORDER BY urn LIMIT ?, ? ) d " +
```
Comment (ericsun2) [inline web/app/dao/DatasetsDAO.java]: This is sufficient for now.

We will add more API to perform lookups for other attributes. Then we can switch to simpler pagenation SQL in the future.

**Comment by ericsun2** [comment]
LGTM

**Comment by jerrybai2009** [comment]
We encountered the ember exception: Uncaught TypeError: Cannot read property 'shouldSupercede' of undefined.

Have to reimplemented the flow routing to walk around

**Comment by ericsun2** [comment]
Ship it

**Pull request merged**